README.md

```diff
@@ -8,4 +8,4 @@ npm start
 ```
 
 - to creatre maze - POST: localhost:3000/maze/create, body: { "maze": [3, 1, [2, [5,[4, 3]]], []] }
-- to find value - POST: localhost:3000/maze/find, body: { "operator": "greater", "value": 2 }
+- to find value - GET: localhost:3000/maze/find?operator=greater&value=2
```

Comment:
Yep, this is better. :)

But overall verbs like `create` and `find` are not allowed in REST APIs. REST uses resources + HTTP VERBS.